### DISCUSSION TITLE
blog/exploring-keystatic/

### DISCUSSION THREAD

**Discussion by giscus** [armno/armno.in.th#40] [Blog Post Comments]
# blog/exploring-keystatic/

Keystatic is a CMS to manage Markdown content. In this blog post, I noted down how I integrate Keystatic to my Astro blog.

https://armno.in.th/blog/exploring-keystatic/

**Comment by armno** [comment]
first comment ever from giscus!

**Comment by proxima812** [comment]
Hello, Armno! Keystatic made it possible to make content on MDX! 

thanks for information about. It helped a lot 🙏
```ts
datePublished: z.union([z.string().datetime(), z.date()]),
``` 


```ts
	content: fields.mdx({
					label: "content",
					description: "",
					options: {
						image: {
							directory: "src/assets/images/posts",
							publicPath: "../../assets/images/posts/",
						},
					},
				}),
```
more about https://docsmill.dev/npm/@keystatic/core@0.5.5#/.fields.mdx

**Reply by armno** [reply]
Thanks for the update 🙏
